automate.py

```diff
@@ -164,6 +167,60 @@ def run_important_traces(self):
                 event.content['message'] = content
                 self._tracer.controller.buffers.app.put(event)
 
+    def schedule_id(self, id_):
+        """Keep track of scheduled ids"""
+        if not isinstance(id_, str):
+            raise AttributeError("Invalid id type, for schedule.")
+        self.ids.add(id_)
+        return id_
+
+    def schedule_traces(self, func, settings_=settings):
```

Comment:
@Alopalao the intention to pass `func` was great here, but `schedule_traces` internally is already assuming settings that are for `self.run_traces` so no need to pass a func, you can assume it'll be be and then the description of the docstring you can update, it's responble for trying to schedule this function and passing the settings. Same things for `schedule_important_traces`

Reply:
Got it. I thought it needed the `self.main` actualization. Updated